MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -187,6 +187,13 @@ - (void)pauseOrResumeMetricsCollectionIfRequired {
 
 - (void)flush {
     if (self.paused) {
+        [self pushDebugEventWithAttributes:@{MMEDebugEventType: MMEDebugEventTypeFlush,
+                                             MMEEventKeyLocalDebugDescription: @"Aborting flushing of event queue because collection is paused."}];
+        
+        if (self.eventQueue.count >= self.configuration.eventFlushCountThreshold) {
```

Comment:
So, we may need to keep the return here to prevent HTTP operations in low power mode, user opt out, and possibly some other reasons that I'm not sure about. I think we're fine with queuing the events while paused though. (I'll remove that extra debug log -- missed that)

Reply:
yes, you want to keep the return, just remove the extra debug event about emptying the queue